modules/bullet/space_bullet.cpp

```diff
@@ -152,6 +152,16 @@ int BulletPhysicsDirectSpaceState::intersect_shape(const RID &p_shape, const Tra
 }
 
 bool BulletPhysicsDirectSpaceState::cast_motion(const RID &p_shape, const Transform &p_xform, const Vector3 &p_motion, float p_margin, float &r_closest_safe, float &r_closest_unsafe, const Set<RID> &p_exclude, uint32_t p_collision_mask, bool p_collide_with_bodies, bool p_collide_with_areas, ShapeRestInfo *r_info) {
+
+	r_closest_safe = 0.f;
```

Comment:
It's a matter of taste but I find fully qualified `0.0f` and `1.0f` more readable.

Reply:
Changed all 0.f and 1.f's to 0.0f and 1.0f.